subprojects/common/report-viewer/src/main/kotlin/com/avito/report/ReportsAddApiImpl.kt

```diff
@@ -53,10 +53,10 @@ internal class ReportsAddApiImpl(private val requestProvider: JsonRpcRequestProv
                         reportCoordinates = reportCoordinates,
                         buildId = buildId,
                         test = test,
-                        status = Status.Lost,
+                        status = null,
```

Comment:
How does it work? How can we send no status?

Reply:
it's decided on backend